apps/cowswap-frontend/src/modules/limitOrders/hooks/useUpdateCurrencyAmount.ts

```diff
@@ -13,21 +14,54 @@ import { useUpdateLimitOrdersRawState } from 'modules/limitOrders/hooks/useLimit
 import { LimitOrdersRawState } from 'modules/limitOrders/state/limitOrdersRawStateAtom'
 
 import { calculateAmountForRate } from 'utils/orderUtils/calculateAmountForRate'
+import { calculateRateForAmount } from 'utils/orderUtils/calculateRateForAmount'
+
+import { limitOrdersSettingsAtom } from '..'
```

Comment:
The import looks weird...

Reply:
Hmm, indeed, will fix it

Reply:
https://github.com/cowprotocol/cowswap/pull/5339